index.html

```diff
@@ -96,62 +139,49 @@ <h4>Multi-Purpose Stories</h4>
-                  <div class="story-left invert">
-                       <div class="desc">
-                           <h4>Multi-Purpose Stories</h4>
-                           <p>A daily selection of privately personalized reads; 
-                               no accounts or sign-ups required. has been the industry's 
-                               standard dummy text ever since the 1500s, when an unknown 
-                               printer took a standard dummy text.
-                           </p>
-                           <div class="skills">
-                               <p>css</p>
-                               <p>html</p>
-                               <p>bootstrap</p>
-                               <p>Ruby</p>
-                           </div>
-                           <button type="button">See Project</button>
-                       </div>
-                       <div class="rect"></div>
-                   </div>
-           </div>
+                    <div class="desc">
+                        <h4>Multi-Purpose Stories</h4>
+                        <p>A daily selection of privately personalized reads;
+                            no accounts or sign-ups required. has been the industry's
+                            standard dummy text ever since the 1500s, when an unknown
+                            printer took a standard dummy text.
+                        </p>
+                        <div class="skills">
+                            <p>css</p>
+                            <p>html</p>
+                            <p>bootstrap</p>
+                            <p>Ruby</p>
+                        </div>
+                        <button type="button">See Project</button>
+                    </div>
+                    <div class="rect"></div>
+                </div>
+            </div>
+        </div>
+    </section>
 
-       </div>
-   </section>
+    <section class="contact">
+        <div class="contact-text">
+            <h5 class="chat">Lets's Chat!</h5>
+            <h5 class="mobile">CONTACT US</h5>
+            <p>If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="contact" class="contact-form">
+            <div class="data">
+                <input type="text" class="contact-element" name="user-fname" placeholder="First Name"><br class="custom">
+                <input type="text" class="contact-element" name="user-lname" placeholder="Last Name">
+            </div>
+            <label for="email"></label>
+            <input type="email" id="email" class="contact-element1" name="user-email" placeholder="Email Address"><br>
+            <textarea name="user-message" class="contact-element2" cols="30" rows="10" placeholder="Write your message here"></textarea><br><br>
+            <button type="button" class="contact-button">Seen</button>
```

Comment:
Change the button as shown below
#### Original design
![right](https://user-images.githubusercontent.com/58376020/125502711-b4d27a70-07c7-425b-8e9f-f7297e5752f9.png)

#### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/125502738-e25318f2-daee-4038-96fb-795dee7d24cb.png)